torch/tensors/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .uint1_7 import UInt4Tensor
```

Comment:
Use `torch._subclasses`, it already exists

Reply:
we need more complete implementations than that I think